mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_TileManager.java

```diff
@@ -33,7 +33,7 @@ public WebResourceResponse getTile(String url) throws Exception {
 
         /*
          * now get the file path...example path:
-         * /storage/emulated/0/Android/data/org.exarhteam.iitc_mobile/files/mqcdn.com/tiles/1.0.0/map/18/137397/89580.jpg
+         * /storage/emulated/0/Android/data/org.exarhteam.iitc_mobile_px/files/mqcdn.com/tiles/1.0.0/map/18/137397/89580.jpg
```

Comment:
No need to change the string

Reply:
Is a relict so i can install official version and this test side by side